libr/util/sandbox.c

```diff
@@ -15,6 +15,10 @@
 #include <priv.h>
 #endif
 
+#ifdef __MINGW32__
+#include <unistd.h>
```

Comment:
maybe worth adding it in r_types.h, so other files can be affected

Reply:
the error comes again after installing in r_types.h

Reply:
Done